### PR TITLE
US-513230: Update Hazelcast CloudK/CMC Documentation

### PR DESCRIPTION
**Pull request by Pushpendrak18**
- Updated pega README under hazelcast section
- Updated Deploying-Pega-on-EKS.md runbook

### CODE REVIEW

**Comment by Pushpendrak18** [comment]
> Hello, Just caught a typo in the EKS runbook:
> 
> 558 Either to enable Hazelcast in client-server model and configure the number of replicas and username & passowrd for authentication --> please change it to "and password".
> 
> Also, I think it would be beneficial to add the image distinction between Pega Platform versions 8.6-8.7 and 8.8 as comments before the parameters in the Helm chart itself.
> 
> Otherwise, it looks great! Thanks.

**Comment by Pushpendrak18** [comment]
Updated:

- Fixed the typo
- Added comments for image distinction for Pega Plateform 8.6-8.7 and 8.8+

**Comment by Pushpendrak18** [comment]
Closed by mistake

**Review by kingakowalska1 — APPROVED** [review]
Looks great, thanks for the changes!

**Review by nanil4568 — APPROVED** [review]

**Review by kishorv10 — APPROVED** [review]

**Review by taz-pega-work — COMMENTED** [review]
@Pushpendrak18 - I see that you made changes to the EKS runbook in this PR. Was this made because the other 4 runbooks had the correct wording and EKS was missing? Or did you not make this change in the other 4 runbooks.
Please clarify if all of the runbooks are updated to describe the Hazelcast service and image appropriately.

**Comment by Pushpendrak18** [comment]
> @Pushpendrak18 - I see that you made changes to the EKS runbook in this PR. Was this made because the other 4 runbooks had the correct wording and EKS was missing? Or did you not make this change in the other 4 runbooks. Please clarify if all of the runbooks are updated to describe the Hazelcast service and image appropriately.

Made changes in other 4 notebooks

File: docs/Deploying-Pega-on-AKS.md [hunk 1]
```diff
@@ -510,8 +510,8 @@ helm inspect values pega/pega > <local filepath>/aks-demo/pega.yaml
...
-    | hazelcast.image:        | Specify the Pega-provided `clustering-service` Docker image that you downloaded and pushed to your Docker registry. | Image: "\<Registry host name:Port>/my-pega-installer:\<Pega Platform version>" |
-    | hazelcast.enabled: hazelcast.replicas: hazelcast.username: hazelcast.password: | Either to enable Hazelcast in client-server model and configure the number of replicas and username & passowrd for authentication | <ul><li>enabled: true/false <br/> Set to true if you want to deploy pega platform in client-server Hazelcast model, otherwise false. *Note: Set this value as false for Pega platform versions below 8.6, if not set the installation will fail.* </li><li>replicas: <No. of initial server members to join(3 or more based on deployment)> </li><li>username: "\<UserName for authentication\>" </li><li> password: "\<Password for authentication\>" </li></ul> |
+    | hazelcast.image: hazelcast.clusteringServiceImage:       | Specify the Pega-provided clustering-service Docker image that you downloaded and pushed to your Docker registry. | Image: "<Registry host name:Port>/my-pega-installer:<Pega Platform version>" |
```
Comment (taz-pega-work) [inline docs/Deploying-Pega-on-AKS.md:513]: Thanks for adding the other runbooks ;-) Please tweak the description a bit and use proper punctuation>>
    | hazelcast.image: hazelcast.clusteringServiceImage:       | Specify the Pega-provided clustering-service Docker image that you downloaded and pushed to your Docker registry. | Image: "<Registry host name:Port>/my-pega-installer:<Pega Platform version>" |
    | hazelcast.enabled: hazelcast.clusteringServiceEnabled: hazelcast.replicas: hazelcast.username: hazelcast.password: | Enable the use of Hazelcast in client-server model for the version of Pega Platform, configure the number of replicas and set a username and password for authentication. | <ul><li>enabled: true/false <br/> Setting below to true will deploy Pega Platform using a client-server Hazelcast model for version 8.6 through 8.7.x. Note: Make sure to set this value as "false" if your Pega Platform version is earlier than "8.6"; if not set, the installation will fail. </li><li>clusteringServiceEnabled: true/false <br/> Setting below to true will deploy Pega Platform using a client-server Hazelcast model for Pega Platform version 8.8 and later. </li><li>replicas: <No. of initial server members to join (use 3 or more based on your deployment size)> </li><li>username: "<UserName for authentication>" </li><li> password: "<Password for authentication>" </li></ul> |

**Review by taz-pega-work — CHANGES_REQUESTED** [review]
I made a few changes to the content and punctuation...please use the guidance across all 5 runbooks in that spot.

File: docs/Deploying-Pega-on-AKS.md (hunk 1 above)
Comment (Pushpendrak18) [reply docs/Deploying-Pega-on-AKS.md]: Hi @taz-pega-work 
I made the requested changes.

**Review by taz-pega-work — APPROVED** [review]
thanks for making those changes in all 5 runbooks. This is awesome! thanks again @Pushpendrak18 !!!!

**Pull request merged**